### PR TITLE
docstring added

### PR DESCRIPTION
**Pull request by ryanbelt**
Table: auto_set_column_width not working [#6047](https://github.com/matplotlib/matplotlib/pull/6047)

doc string has added.
**method description**
```python
""" Given column indexs in either List, Tuple or int. Will be able to
       automatically set the columns into optimal sizes.
```
**method example**
```python
Args:
    col(List): list of indexs
    >>>table.auto_set_column_width([-1,0,1])

    col(Tuple): tuple of indexs
    >>>table.auto_set_column_width((-1,0,1))

    col(int): index integer
    >>>table.auto_set_column_width(-1)
    >>>table.auto_set_column_width(0)
    >>>table.auto_set_column_width(1)
"""
```

### CODE REVIEW

**Comment by QuLogic** [comment]
Please be sure to write [clear commit messages](http://chris.beams.io/posts/git-commit/).

docstring added _to what?_

File: lib/matplotlib/table.py [hunk 1]
```diff
@@ -409,7 +409,21 @@ def _do_cell_alignment(self):
             cell.set_y(bottoms[row])
 
     def auto_set_column_width(self, col):
+        """ Given column indexs in either List, Tuple or int. Will be able to
+        automatically set the columns into optimal sizes.
 
+        Args:
```
Comment (tacaswell) [inline lib/matplotlib/table.py]: Can you please use the numpydoc formating?

File: lib/matplotlib/table.py [hunk 2]
```diff
@@ -409,7 +409,21 @@ def _do_cell_alignment(self):
...
+            col(List): list of indexs
+            >>>table.auto_set_column_width([-1,0,1])
+
+            col(Tuple): tuple of indexs
+            >>>table.auto_set_column_width((-1,0,1))
+
+            col(int): index integer
+            >>>table.auto_set_column_width(-1)
```
Comment (tacaswell) [inline lib/matplotlib/table.py]: What is the meaning of these integers?

**Comment by ryanbelt** [comment]
I have added the details on explain what is the -1,0,1 meaning for before the example.

Thank you.

**Comment by ryanbelt** [comment]
unknown CI server failure. compare to d86e805. I have nothing change on the code excepted the docstring pep8. even on 92ccded, all test cases are passed except pep8.

I need some help on this. Any comment on this?

Thank you

**Comment by tacaswell** [comment]
We have a couple of transient test failures that have not been tracked down and squashed yet.  When we went to 0 tolerance on a majority of the test images we got _most_ of them but it exposed some others.

**Comment by tacaswell** [comment]
Merging as is.

It would be better if it followed the https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt format, but anything is better than nothing.

**Pull request merged**